single_level.py

```diff
@@ -27,14 +27,59 @@ def get_note_types(resource):
 		else:
 			note_types.append(note["jsonmodel_type"])
 	return note_types
+	
+def get_extent_number(resource):
+	extent_number = []
+	for extent in resource["extents"]:
+		if extent["number"] > 0:
+			extent_number.append("true")
+		else:
+			extent_number.append("false")
+	return extent_number
+	
+def get_language(resource):
+	d = resource
+	lang = []
+	if 'language' in d:
+		return "true"
+	else:
+		return "false"
 
+def get_creator(resource):
+	creator_type = []
+	for creator in resource["linked_agents"]:
+		if creator["role"] == ("creator"):
+			creator_type.append("true")
+		else:
+			creator_type.append("false")
+	return creator_type
+	
 def makeRow(resource):
 	global row
 	row = []
+	language_list = get_language(resource)
```

Comment:
I would probably call `language_list` something different because there should only ever be one value (either true or false).